src/inputBackend/input/DevInput.hpp

```diff
@@ -34,15 +34,14 @@ class DevInputBase
 
 	XenBackend::Log mLog;
 	std::string mName;
+	int mFd;
```

Comment:
Implement function
int getMFd()condt{return mFd) 
to have an access to member mFd
Otherwise, it is possible to change the variable from valid value to invalid and base class can not process it (close a file) properly